airbyte-integrations/bases/base-normalization/unit_tests/resources/edge_cases_catalog_expected_nested.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
is this file being used?

Reply:
the test was originally made for testing nested streams in the catalog but in this scenario, it's not using any nesting so it needs an empty nested file for expected nested streams

So for the test to pass, it is needed yes